### PR TITLE
Add the module purchase_cancel_reason. Copy/past from the module sale_cancel_reason (OCA/sale-workflow) developed by CampToCamp.

### PR DESCRIPTION
**Pull request by sylvainvh**

### CODE REVIEW

**Comment by sylvainvh** [comment]
Hello, 
There is a problem with Travis. 
But for the rest everyhing seems good !

**Comment by LoisRForgeFlow** [comment]
It is affecting more PRs and it is related to https://github.com/OCA/maintainer-quality-tools/issues/467

File: purchase_cancel_reason/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Copyright 2013 Guewen Baconnier, Camptocamp SA
+# Copyright 2017 Okia SPRL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{'name': 'Purchase Cancel Reason',
+ 'version': '10.0.1.0.0',
+ 'author': "Camptocamp,Odoo Community Association (OCA), Okia SPRL",
+ 'category': 'Purchase',
+ 'license': 'AGPL-3',
+ 'complexity': 'normal',
+ 'website': "http://www.camptocamp.com",
```
Comment (MiquelRForgeFlow) [inline purchase_cancel_reason/__manifest__.py]: `https://github.com/OCA/purchase-workflow`

File: purchase_cancel_reason/model/purchase_order.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 Guewen Baconnier, Camptocamp SA
+# Copyright 2017 Okia SPRL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields
```
Comment (MiquelRForgeFlow) [inline purchase_cancel_reason/model/purchase_order.py]: alphabetical order

File: purchase_cancel_reason/wizard/purchase_cancel_reason.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2013 Guewen Baconnier, Camptocamp SA
+# Copyright 2017 Okia SPRL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields, api
```
Comment (MiquelRForgeFlow) [inline purchase_cancel_reason/wizard/purchase_cancel_reason.py]: alphabetical order

**Review by MiquelRForgeFlow — APPROVED** [review]
:green_apple: 
code review :+1:

**Comment by sylvainvh** [comment]
@mreficent , modification done !

**Comment by sylvainvh** [comment]
Small UP nobody can validate my PR ? Thank you !

**Comment by sylvainvh** [comment]
New up ?

**Review by yvaucher — APPROVED** [review]

**Pull request merged**